packages/blueprint/package.json

```diff
@@ -4,18 +4,18 @@
   "version": "1.0.2",
```

Comment:
```suggestion
  "version": "1.0.3",
```
Should we bump versions on changes to dependencies?

Reply:
depends, if the changed dependency effect the package so much is validates bumping its version I'd say yes :)

Reply:
But if you look closely, no dependencies has actually changed, only reordered.
yarn...smh